packages/eslint-plugin/docs/rules/space-infix-ops.md

```diff
@@ -0,0 +1,18 @@
+# This rule is aimed at ensuring there are spaces around infix operators. (`space-infix-ops`)
+
+This rule extends the base [`eslint/space-infix-ops`](https://eslint.org/docs/rules/space-infix-ops) rule.
```

Comment:
you probably want to mention it adds support for enums etc. to indicate why the extension rule is useful over the base one.